### PR TITLE
✨ Add endpoint to return tier

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/dto/ErrorResponse.kt [hunk 1]
```diff
@@ -1,7 +0,0 @@
-package uk.gov.justice.digital.hmpps.probationtiering.dto
```
Comment (astroash) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/dto/ErrorResponse.kt:1]: How come this file was removed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/jpa/entity/TierCalculationEntity.kt [hunk 2]
```diff
@@ -22,12 +22,12 @@ data class TierCalculationEntity(
         val id : Long? = null,
 
         @Column
-        val crn: String,
+        val crn : String? = null,
```
Comment (astroash) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/jpa/entity/TierCalculationEntity.kt]: How come `crn` could be null? Are we using any other user identifiers to get their data?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/dto/ErrorResponse.kt (hunk 1 above)
Comment (nickmcmahon01) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/dto/ErrorResponse.kt:1]: we don't use it.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/jpa/entity/TierCalculationEntity.kt (hunk 2 above)
Comment (nickmcmahon01) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/probationtiering/jpa/entity/TierCalculationEntity.kt]: This was addressed in the other PR, it isn't now, it was a convenience thing like these for testing with succinct constructors: https://github.com/ministryofjustice/hmpps-assessments-api/tree/main/src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities

**Review by astroash — APPROVED** [review]

**Pull request merged**